tests/kernel/fifo/fifo_timeout/src/main.c

```diff
@@ -158,6 +158,7 @@ static int test_multiple_threads_pending(struct timeout_order_data *test_data,
 		struct timeout_order_data *data =
 			k_fifo_get(&timeout_order_fifo, K_FOREVER);
 
+		zassert_true(data != NULL, "");
```

Comment:
Wouldn't it be better to use `NULL` for an empty message (in order to align with other changes)?

Reply:
yes, missed that.